assets/js/show-task.js

```diff
@@ -32,17 +37,37 @@ function showTask(inputValue, formattedDate, taskList, form, allTasks, id) {
   const DELETE_ICON = icons["DELETE"];
   const deleteButton = createButton("delete", DELETE_ICON, "Delete task");
 
+  const REVERT_ICON = icons["REVERT"];
+  const revertButton = createButton(
+    HIDE_CLASS,
+    REVERT_ICON,
+    "Revert to previous task"
+  );
+
   tools.appendChild(date);
+  tools.appendChild(saveButton);
   tools.appendChild(doneButton);
   tools.appendChild(editButton);
   tools.appendChild(deleteButton);
+  tools.appendChild(revertButton);
```

Comment:
```suggestion
  tools.append(date, saveButton, doneButton, editButton, deleteButton, revertButton)
```

//see https://developer.mozilla.org/en-US/docs/Web/API/Element/append